src/pages/overview/limits.md

```diff
@@ -2,30 +2,26 @@
 title: Usage Limits and Licensing
 ---
 
-# Usage Limits and Licensing
+# Licensing and Usage Limits
 
 
 Adobe PDF Services API licensing is measured per Document Transaction which is based on the initial
 endpoint request (i.e., API call) and the digital output, with each
-operation in the table described below. Content limits apply and are
-provided along with examples for clarity. These limits do not apply to PDF Embed API.
+operation in the table described below. Usage limits apply and are provided along with examples for clarity. These limits do not apply to PDF Embed API.
 
 Paid credentials enable a greater processing quota and require entering
-into a separate written agreement. To learn more, please see our
+into a separate written agreement. To learn more, please see
 [pricing page](/document-services/pricing/#main)
-for purchase options.
-
-Document Transactions for the Operations below resulting in a Document
-will be counted with the following page limits:
+for purchase options.Document Transactions for the Operations below resulting in a Document
+will be counted with the following page limits.
 
 <InlineAlert slots="text"/>
 
```

Comment:
Add this text as part of <InlineAlert> and not above
<InlineAlert slots="text"/>
Document Transactions for the Operations below resulting in a Document will be counted with the following page limits: